locales/en-US/notes.properties

```diff
@@ -1,5 +1,34 @@
-welcomeTitle2=Hello!
-welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It’s that simple.
+welcomeTitle3=Welcome to Firefox Notes!
+welcomeSubtitle3=A simple tool for taking notes while you browse.
+welcomeOpenNotes3=To open Notes:
+# LOCALIZATION NOTE (welcomeWindowsLinuxShortcut3): shortcut is the
+# key combination to open the sidebar
+welcomeWindowsLinuxShortcut3=On Windows, Linux: {shortcut}
+# LOCALIZATION NOTE (welcomeMacShortcut3): shortcut is the
+# key combination to open the sidebar
+welcomeMacShortcut3=On Mac: {shortcut}
+welcomeAccessNotes3=To access your notes from any laptop or desktop computer:
+# LOCALIZATION NOTE (welcomeSyncInfo3): sync is the label on the sync button
+welcomeSyncInfo3=Select {sync} to sign into your Firefox Account.
+# LOCALIZATION NOTE (welcomeFormatText3): `To format your text, use the toolbar or these commands.`
+# This part is in Bold
+welcomeFormatText3=To format your text,
+# LOCALIZATION NOTE (welcomeFormatUse3): `To format your text, use the toolbar or these commands`
+# This part is in normal font weight
+welcomeFormatUse3=use the toolbar or these commands:
```

Comment:
This is not going to work. Languages might need to move the first part of the English string around, and goodbye formatting. Can we have a single string with markup?

```
<b>To format your text,</b> use the toolbar or these commands:
```